content/developer/reference/frontend/javascript_reference.rst

```diff
@@ -707,6 +707,14 @@ Date & Time (`datetime`)
 
             <field name="datetimefield" options="{'rounding': 10}" />
 
+    - `show_seconds`: when set to false, it hides the seconds from the datetime field.
+      The field will still accept datetime values, but the seconds will be hidden in
+      the UI (default: `true`).
+
+        .. code-block:: xml
+
+            <field name="datetimefield" widget="datetime" options="{'show_seconds': false}" />
```

Comment:
Is the widget necessary here?

Reply:
not always but in a list view yes. So I would keep it here